### PR TITLE
Add a icon to the exe file

### PR DESCRIPTION
**Pull request by MGGSK**
Added a icon to the apps exe file.

## Description
Set the exe icon in the project properties to "Assets\Tiles\GalleryIcon.ico"

## Motivation and Context
Don't know why the exe doesn't have an icon.
Fixed this issue: https://github.com/microsoft/WinUI-Gallery/issues/1600

## How Has This Been Tested?
No testing needed.

## Screenshots (if appropriate):
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by karkarl** [comment]
Thanks so much for the fix!

**Review by karkarl — APPROVED** [review]

**Comment by Jay-o-Way** [comment]
can you confirm if this solves #1512?

**Comment by skalsi-pexip** [comment]
TL;DR - this issue is live and needs to be re-opened. I can repo this by downloading WinUI-Gallary from the Store and observing a generic Windows App Icon not the ico in this pr. Here's images:

I am troubleshooting why my WinUI3 App does not have a 16x16 Asset appearing when I have supplied _all_ of the Visual Assets in the package.appxmanifest. 

It seems I'm not alone as the WinUi3 App has the same issue! 

image
image

I did try the code in this PR and it didn't work.
